ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -105,7 +105,7 @@ class DashboardServiceTopology(ServiceTopology):
             ),
             NodeStage(
                 type_=OMetaTagAndClassification,
-                processor="yield_tag",
+                processor="yield_bulk_tags",
```

Comment:
We'll keep a `yield_bulk_tags` method to yield the tags if get them all at once.
Otherwise if we get it attached to the entities we'll use `yield_tags`